src/en/users-models.md

```diff
@@ -112,6 +112,50 @@ bob                   login      1 hour ago    58 minutes ago
 jim                   add-model  2016-11-14    58 minutes ago
 ```
 
+### Controller access for external users
+
+It is possible to give a user access to a controller without creating a local
+account for them. To do this, these criteria must first be met:
+
+- The user must already have an account on an external identity manager,
+  such as the Ubuntu SSO
+- The controller must have been created (bootstrapped) using the identity
+  configuration option, like here where we use the URL for the Ubuntu SSO
+  and Juju: `--config identity-url=https://api.jujucharms.com/identity`
+- The user must first log in to http://jujucharms.com at least once before
+  attempting to log in to Juju as an external user
+- The user must install Juju on the machine from which they will access the
+  controller
+
+On the controller, you grant Frances access to add models using:
+
+```bash
+juju grant frances@external addmodel
+```
```

Comment:
I think @external (and @local?) might be worth a small explanation, as this reader may think this is an arbitrary example rather than a requirement.